src/crimson/common/logfile.h

```diff
@@ -0,0 +1,24 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2011 New Dream Network
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ *
+ */
+
+#pragma once
+
+#include <string_view>
+
+// Write a logfile with the current log, using the configuration in the
+// provided conf structure.
+[[nodiscard]] int logfile_set_ostream(std::string_view log_file);
```

Comment:
As this is marked 'nodiscard' - can you add a comment about the meaning of the returned value?